tutorials/solution_strategies.ipynb

```diff
@@ -0,0 +1,224 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #78.            ) / self.params.get("characteristic_displacement_scaling", 1.0)
</pre><p>Is this different from the characteristic displacement given in the solid material (L108)? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/1208/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Yes. But it was a bug (from the old version of the code).